packages/frontend/src/components/nft/NFTTransferModal.js

```diff
@@ -0,0 +1,397 @@
+import { useDispatch, useSelector } from 'react-redux';
+import styled from 'styled-components';
+
+import { EXPLORER_URL } from '../../config';
+import { checkAccountAvailable } from '../../redux/actions/account';
+import { clearLocalAlert, showCustomAlert } from '../../redux/actions/status';
+import { selectBalance } from '../../redux/slices/account';
+import { actions as nftActions } from '../../redux/slices/nft';
+import NonFungibleTokens, { NFT_TRANSFER_GAS } from '../../services/NonFungibleTokens';
+import isMobile from '../../utils/isMobile';
+import Balance from '../common/balance/Balance';
+import FormButton from '../common/FormButton';
+import Modal from '../common/modal/Modal';
+import ModalFooter from '../common/modal/ModalFooter';
+import SafeTranslate from '../SafeTranslate';
+import ReceiverInputWithLabel from '../send/components/ReceiverInputWithLabel';
+import AvatarSuccessIcon from '../svg/AvatarSuccessIcon';
+import EstimatedFees from '../transfer/EstimatedFees';
+
+
+const StyledContainer = styled.div`
+    &&& {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        padding: 0 0 0 0;
+
+        img {
+            width: 100%  ;
+            max-width: 472px;
+            border-radius: 8px;
+            margin-bottom: 16px;
+        }
+
+        .transfer-txt {
+            margin-top: 16px;
+        }
+
+        .confirm-txt {
+            margin-bottom: 4px  ; 
+        }
+
+        .confirm-img {
+            width: 272px;
+        }
+
+        .confirm-nft-card {
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+            position: relative;
+
+            width: 100%;
+            margin-top: 16px;
+
+            background: #FFFFFF;
+            box-shadow: 0px 45px 56px rgba(0, 0, 0, 0.07), 0px 10.0513px 12.5083px rgba(0, 0, 0, 0.0417275), 0px 2.99255px 3.72406px rgba(0, 0, 0, 0.0282725);
+            border-radius: 8px;
+
+            .from-box {
+                position: relative;
+                width: 100%;
+                height: 74px;
+                border-top: 1px solid #F0F0F1;
+            }
+
+            .to-box {
+                position: relative;
+                width: 100%;
+                height: 74px;
+                border-top: 1px solid #F0F0F1;
+            }
+
+        }
+
+        .confirm-txt {
+            font-size: 14px;
+            line-height: 150%;
+            color: #72727A;
+            margin-left: 16px  ;
+        }
+
+
+        .h-right {
+            position: absolute;
+            right: 16px;
+            text-align: right;
+        }
+
+        .estimate-fee-card {
+            display: flex;
+            flex-direction: column;
+            position: relative;
+
+            width: 100%;
+            height: 78px;
+            margin-top: 16px;
+
+            border: 1px solid #F0F0F1;
+            box-sizing: border-box;
+            border-radius: 8px;
+        }
```

Comment:
```suggestion
        .estimated-fees {
            width: 100%;
            margin-top: 20px;
        }
```

Shouldn't apply same style twice:) See comment: https://github.com/near/near-wallet/pull/2205/files#r769122247